setup.cfg

```diff
@@ -14,7 +14,8 @@ max-line-length = 119
 # E133: closing bracket missing indentation
 # E226: missing whitespace around arithmetic operator
 # W503: line break occurred before a binary operator
-ignore = E133,E226,W503
+# W504: line break occurred after a binary operator
+ignore = E133,E226,W503,W504
```

Comment:
Wouldn't it be better to reformat the code that triggers W504 rather than ignoring it? IIRC in Rubicon we usually put line breaks before the operators, which is why W503 is ignored. I'm not sure why flake8 is detecting these W504 issues only now - perhaps they changed something in an update.

Reply:
Indeed the flake8 did change something in the update so that when you have add ___ignore___ argument the both w503 and w504 are enabled.

Reply:
W504 fixes made and this change reverted.